code/modules/projectiles/guns/smartgun.dm

```diff
@@ -30,13 +30,19 @@
 		/datum/action/item_action/smartgun/toggle_accuracy_improvement,
 		/datum/action/item_action/smartgun/toggle_ammo_type,
 		/datum/action/item_action/smartgun/toggle_auto_fire,
+		/datum/action/item_action/smartgun/toggle_frontline_mode,
 		/datum/action/item_action/smartgun/toggle_lethal_mode,
 		/datum/action/item_action/smartgun/toggle_motion_detector,
 		/datum/action/item_action/smartgun/toggle_recoil_compensation,
 	)
-	var/datum/ammo/ammo_primary = /datum/ammo/bullet/smartgun //Toggled ammo type
-	var/datum/ammo/ammo_secondary = /datum/ammo/bullet/smartgun/armor_piercing //Toggled ammo type
+	var/datum/ammo/ammo_primary //Toggled ammo type
+	var/datum/ammo/ammo_secondary //Toggled ammo type
+	var/datum/ammo/ammo_primary_def = /datum/ammo/bullet/smartgun
+	var/datum/ammo/ammo_secondary_def = /datum/ammo/bullet/smartgun/armor_piercing
+	var/datum/ammo/ammo_primary_alt = /datum/ammo/bullet/smartgun/alt
+	var/datum/ammo/ammo_secondary_alt = /datum/ammo/bullet/smartgun/armor_piercing/alt
 	var/iff_enabled = TRUE //Begin with the safety on.
+	var/frontline_enabled = FALSE //Begins with 'normal IFF' enabled (shoots through friendly marines, instead of Alt-IFF)
```

Comment:
add a code doc as to what this variable does, which is like
```
/// This controls ... 
var/whatever
```

Reply:
bon'd and appetite'd

Reply:
Theres a difference between `//` and `///`
See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/AUTODOC.md

You only did `//` when you needed to do `///`